src/hooks/embalmerFacet/useRewrapSarcophagus.ts

```diff
@@ -14,24 +14,29 @@ export function useRewrapSarcophagus(sarcoId: string, resurrectionTime: Date | n
 
   const timeInSeconds = resurrectionTime ? Math.trunc(resurrectionTime.getTime() / 1000) : 0;
 
-  const { config } = usePrepareContractWrite({
+  const { config, isError } = usePrepareContractWrite({
```

Comment:
Something I recently realised - while `isError` from `usePrepareContractWrite` will be true if the transaction *will* fail, a completely different `isError` is returned from `useContractWrite` if the transaction *actually* fails. I believe this means, `usePrepareContractWrite` would not catch a "failure" where the user cancels the request.

Check out `useCleanSarcophagus` for how I ended up using and exposing both of these `isError`s.